services/app/src/components/base/Card.tsx

```diff
@@ -137,9 +141,9 @@ class Card extends React.Component<Props, {}> {
             </Menu>
           </span>
           <div className="title">{this.props.title}</div>
-        </div>
-        {this.props.header && <div className="header">{this.props.header}</div>}
-        <div className="article">
+        </div>;
```

Comment:
Weird, should there be a semicolon here?

Reply:
wierd, yeah I will remove these semicolons.